resources/scripts/HashValueGenerator.py

```diff
@@ -13,18 +15,19 @@ def createHashValue(img):
     return createdHash
 
 
-with open(sys.argv[1])) as inputJson:
+with open(sys.argv[1]) as inputJson:
     image_data = json.load(inputJson)
 
 id = image_data['image_id']
 imageBytes = image_data['image_as_byte_string']
 
-with Image.open(io.BytesIO(imageBytes))as img:
+with Image.open(io.BytesIO(base64.b64decode(imageBytes))) as img:
     createdHash = createHashValue(img)
 
+
 returnData = {
     'id': id,
-    'hash': createdHash
+    'hash': str(createdHash)
 }
 
 with open(sys.argv[2], 'w') as f:
```

Comment:
You could also print the hash to stdout, as it has a fixed length. This way you don't need an additional output file.

Reply:
This is a good idea, i will implement it